engines/sci/engine/kgraphics.cpp

```diff
@@ -1210,6 +1212,22 @@ reg_t kShakeScreen(EngineState *s, int argc, reg_t *argv) {
 	return s->r_acc;
 }
 
+Common::String textKeeper[10];
```

Comment:
Why is this needed at all?

Reply:
Thanks for taking a look. 
It's for the text that appear in the question book of Laurabow2 (shown below). It's here to avoid passing the wrong/repeated text into ttsMan. The text that gets printed every time a new option gets highlighted is regularly more than the option itself.
<img width="637" alt="Screen Shot 2564-08-11 at 22 19 35" src="https://user-images.githubusercontent.com/77198777/129046054-dcf8d39f-a5a0-4ca9-bb32-d04b0400a9c7.png">